localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -410,6 +410,9 @@
         <trans-unit id="msgRefreshTokenError">
             <source xml:lang="en">Error when refreshing token</source>
         </trans-unit>
+        <trans-unit id="msgAzureCredStoreSaveFailedError">
+            <source xml:lang="en">Keys for token cache could not be saved in credential store, this may cause Azure access token persistence issues and connection instabilities. It's likely that SqlTools has reached credential storage limit on Windows, please clear at least 2 credentials created by Microsoft.SqlTools in Windows Credential Manager and reload.</source>
```

Comment:
Little clearer, there isn't actually a "created by" field or anything which might confuse people.

```suggestion
            <source xml:lang="en">Keys for token cache could not be saved in credential store, this may cause Azure access token persistence issues and connection instabilities. It's likely that SqlTools has reached credential storage limit on Windows, please clear at least 2 credentials that start with "Microsoft.SqlTools|" in Windows Credential Manager and reload.</source>
```